info_processing/code_processing/enry_processor.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict
+
+from enry import get_language_by_extension
+
+using_languages = ['python', 'java', 'javascript']
```

Comment:
it's a constant - `USING_LANGUAGES` (TBH - it's better to call `ALLOWED_LANGS`)
optional